cmd/server/assets/realmadmin/_stats_keyserver.html

```diff
@@ -107,7 +107,6 @@ <h5 class="modal-title">TEK publish metrics</h5>
         <strong>Total # TEKs published</strong>
         <p>
           This line tracks the total number of TEKs published to the key-server.
-          This is expected to be similar to the number of tokens claimed on the verification server.
```

Comment:
@mikehelmick do we have anything special/further to say here

Reply:
not at the moment.